src/main/java/com/notably/model/person/Path.java

```diff
@@ -0,0 +1,30 @@
+package com.notably.model.person;
```

Comment:
Let's change the package to `com.notably.model.block` instead?
In addition, we'll need to move this file to the `com/notably/model/block` folder

Reply:
Ok i've renamed the Person folder to Block folder.